superset-frontend/src/explore/App.jsx

```diff
@@ -60,4 +39,4 @@ const App = () => (
   </Provider>
 );
 
-export default hot(module)(App);
+export default hot(App);
```

Comment:
Move store creation out of `App.jsx` to avoid this error during hot reload:

> `<Provider>` does not support changing `store` on the fly. It is most likely that you see this error because you updated to Redux 2.x and React Redux 2.x which no longer hot reload reducers automatically. See https://github.com/reduxjs/react-redux/releases/tag/v2.0.0 for the migration instructions.